### PR TITLE
data: d4community

### PR DESCRIPTION
**Pull request by D4-Community**
As discussed in previous PRs, creating this BioDrop profile for the GitHub Community account.

### CODE REVIEW

**Review by Dun-sin — APPROVED** [review]
Looks good to me

**Review by sital002 — COMMENTED** [review]
Your GitHub username must match your filename instead of ```data/d4community.json```
it should be ```data/D4-Community.json```.
Please make the necessary changes to get your profile merged.

**Comment by Sagar0-0** [comment]
D4Community usename is already assigned to our community organization and we can not create a PR from there. So, please allow this username so that we can have consistent url for biodrop as in our other platforms.

**Review by eddiejaoude — DISMISSED** [review]
Usernames must match, otherwise various automation won't work - we will have the ability to add additional profiles with a non matching username later

**Comment by D4-Community** [comment]
Anything else I gotta do?

File: data/D4-community.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      "name": "Discord server",
+      "url": "https://discord.gg/BvBTbk43Cx",
+      "icon": "FaDiscord"
+    },
+    {
+      "group": "Connect",
+      "name": "Instagram Page",
+      "url": "https://www.instagram.com/d4community/",
+      "icon": "FaInstagram"
+    },
```
Comment (SaraJaoude) [inline data/D4-community.json:55]: remove comma:
```suggestion
    }
```

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please make sure your file name matches your GitHub username **exactly** (it is case sensitive). Instead of `data/D4-community.json` it should be`data/D4-Community`. You need to change this for a capital C.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**